refarch-integrations/refarch-email-integration/refarch-email-integration-rest-example/src/main/java/de/muenchen/refarch/email/integration/EmailRestExampleApplication.java

```diff
@@ -0,0 +1,23 @@
+package de.muenchen.refarch.email.integration;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.context.event.ApplicationReadyEvent;
+import org.springframework.context.event.EventListener;
+
+@SpringBootApplication
+public class EmailRestExampleApplication {
+    @Autowired
+    TestService testService;
+
+    public static void main(final String[] args) {
+        SpringApplication.run(EmailRestExampleApplication.class, args);
+    }
+
+    @EventListener(ApplicationReadyEvent.class)
+    void sendTestMail() {
+        this.testService.testSendMail();
+        System.exit(0);
+    }
```

Comment:
why do we need that?

Reply:
Just for having a runnable example and for e2e testing.
Could theoretically be done with testcontainers but with this there is also a real usage example.
Would let it this way for the moment.